src/addListeners.js

```diff
@@ -5,7 +5,11 @@ var validate = require('./validate');
 var getErrorElement = require('./helpers/getErrorElement');
 
 // Handle elements with data-validations properties
-$(document).on('keyup click change blur', 'input, textarea, select', function (e) {
+$(document).on('keyup click change blur', 'input, textarea, select', onChange);
+$(document).on('blur keyup', '[data-validations]', onChange);
+
+function onChange(e) {
+  var $input = $(e.target);
```

Comment:
Yes, I reconciled the 2 event handlers but tried not to change the code until I have time to properly look through it.